cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -71,6 +70,7 @@ var (
 // (and thus with different runtime data struct, all of them requested to be compliant to this interface)
 type certsData interface {
```

Comment:
btw, unless i'm mistaken allCertsLongDesc and allCertsExample are unused in this file.

Reply:
done